scripts/install.sh

```diff
@@ -56,6 +56,19 @@ fail_install() {
   exit 1
 }
 
+verify_latest_release_download() {
```

Comment:
I feel this function might be overkill as we can just use its code inline where you invoke it (L101).

Reply:
Thanks for the review! To be clear, you would prefer to put the _if_ _condition_ (L60) outside a function and in the main bash script, i.e. immediately after the `$latest` variable is created? (L98)

Reply:
yup